### PR TITLE
codeowners tweaks

### PR DESCRIPTION
**Pull request by clux**
for #184 but also standardising output a bit better.

### CODE REVIEW

File: shipcat_cli/src/get.rs [hunk 1]
```diff
@@ -51,15 +51,13 @@ pub fn codeowners(conf: &Config) -> Result<Vec<String>> {
         // teams guaranteed by validates on Manifest and Config
         if let Some(t) = &conf.teams.iter().find(|t| t.name == md.team) {
             if let Some(gha) = &t.githubAdmins {
-                ghids.push(format!("@{}/{}", org, gha));
+                ghids.push(format!("@{}/{}", org.to_lowercase(), gha));
```
Comment (janslow) [inline shipcat_cli/src/get.rs]: Should `gha` be lowercased as well?

File: shipcat_cli/src/get.rs [hunk 2]
```diff
@@ -51,15 +51,13 @@ pub fn codeowners(conf: &Config) -> Result<Vec<String>> {
...
             }
             // TODO: take this out now that we have teams?
             for o in t.owners.clone() {
                 ghids.push(format!("@{}", o.github.unwrap()));
             }
-            if !t.owners.is_empty() {
-                output.push(format!("services/{}/* {}", mf.name, ghids.join(" ")));
-            }
+            output.push(format!("services/{}/* {}", mf.name, ghids.join(" ")).trim().into());
```
Comment (janslow) [inline shipcat_cli/src/get.rs]: What if `ghids` is empty (e.g., `owners` is empty and `githubAdmins` is none)?

File: shipcat_cli/src/get.rs (hunk 1 above)
Comment (clux) [reply shipcat_cli/src/get.rs]: that's already enforced due to vault policy reconcile

File: shipcat_cli/src/get.rs (hunk 2 above)
Comment (clux) [reply shipcat_cli/src/get.rs]: have fixed that now.

**Pull request merged**